pkg/application/application.go

```diff
@@ -221,3 +222,26 @@ func Exists(client *occlient.Client, name string) (bool, error) {
 	}
 	return false, nil
 }
+
+func NamespaceComponent(client *occlient.Client, componentName string) (string, error) {
+
+	// Error if it's blank
+	if componentName == "" {
+		return "", errors.New("namespacing: component name cannot be blank")
+	}
+
+	// Get the current application
+	currentApplication, err := GetCurrent(client)
```

Comment:
The problem is that not all functions that need the component namespaced have the currentApplication. So instead of just plugging in the function, we'd have to enter 5 more lines of code getting the current application. That's why I added GetCurrent to this.

Reply:
I know, we have to update those function :-( 
The question is if we do it in this PR, or create another one.